src/app/cheat-sheets/game-base/space-age/space-age.component.html

```diff
@@ -140,21 +145,21 @@ <h5>Planet Start</h5>
       </li>
 
       <li>
-        Space is can be limited, and scrap recycling yield a lot of products, thus using robots can be very convenient to sort everything
-        out, and craft starter items
+        Space is almost always limited and scrap recycling yields a lot of products, thus using robots can be very convenient to sort everything
+        out and craft starter items.
       </li>
 
       <li>
         Setup filters on <app-factorio-icon [icon]="dataService.getFactorioIcon('Storage_chest', 12)"></app-factorio-icon> (12 recycled
         items from scrap) + 1 generic storage to start
       </li>
       <li>
-        Use
+        Use 1 each of
```

Comment:
I dont think saying 1 each is necessary here, imo keeping it general is fine
```suggestion
        Use
```